be/src/storage/olap_type_infra.h

```diff
@@ -0,0 +1,125 @@
+// This file is made available under Elastic License 2.0.
```

Comment:
// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.

Reply:
done